tiledb/sm/array/consistency.h

```diff
@@ -110,27 +111,34 @@ class ConsistencyController {
   /**
    * Wrapper around a multimap registration operation.
    *
-   * Note: this function is private bacause it may only be called by the
+   * Note: this function is private because it may only be called by the
    * ConsistencySentry constructor.
    *
    * @pre the given URI is the root directory of the Array and is not empty.
+   * @pre until array openness and registration are fully atomic, this
+   * function may not request data from an array that is not yet fully opened.
```

Comment:
Strictly speaking, this isn't a precondition to call the function, but rather a limitation on the implementation. I'd remove the `@pre` annotation.